docs/source/user_guide/automl.ipynb

```diff
@@ -657,7 +657,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "automl.results"
+    "import pprint\n",
+    "pp = pprint.PrettyPrinter(indent=0, width=100, depth=3, compact=True, sort_dicts=False)\n",
```

Comment:
could we set `depth=2` and then showcase just one pipelines results with:
`automl.results['pipeline_results'][0]`?  I still think the output is rather long and takes multiple scrolls to get through.

Reply:
My main worry is that `depth=2` gives users very little information:
<img width="621" alt="image" src="https://user-images.githubusercontent.com/22552445/153952454-58b11c39-884d-4604-a6dd-64e9d3f99dac.png">

I can definitely try to combine it with your latter suggestion and see how it works!

Reply:
ah true that doesn't look the best.. lmk if adding more information will make it better but def not blocking!

Reply:
<img width="808" alt="image" src="https://user-images.githubusercontent.com/22552445/153995226-482b0b68-6b9d-4e16-84cb-58a84c7e931a.png">

I like the previous better. will change back!